quantum/quantum_keycodes.h

```diff
@@ -597,9 +597,13 @@ enum quantum_keycodes {
 
     QK_MAKE,
 
-    // Caps Word
-    CAPS_WORD,
+    SECURE_LOCK,
+    SECURE_UNLOCK,
+    SECURE_TOGGLE,
 
+      // Caps Word
+    CAPS_WORD,
```

Comment:
Is that comment really necesary? Seems obvious and redundant with the macro name

Reply:
You're right, this comment is entirely redundant. I'll remove it.